tracee/tracee.go

```diff
@@ -29,6 +29,9 @@ type TraceeConfig struct {
 	SecurityAlerts     bool
 	maxPidsCache       int // maximum number of pids to cache per mnt ns (in Tracee.pidsInMntns)
 	BPFObjPath         string
+	PinObjectType      string
```

Comment:
Please use a "config" struct for all of these three (same as done for Capture and Output above).
I also think that this should be a slice of pin configs so we can support more pinning options in the future:

```
type TraceeConfig struct {
    ...
    BpfPinnings    []BpfPinConfig
}

type BpfPinConfig struct {
    ObjectType    string
    ObjectName    string
    Path          string
}
```